stock_picking_location_check/readme/CONTEXT.md

```diff
@@ -0,0 +1,4 @@
+Users sometimes change locations (location_id/location_dest_id) in the picking after
```

Comment:
In standard, it is not possible to select a location (dest  or source) that is not the same or child as the picking one (replicated on stock.move level) and then on move lines.

Do you have particular addons that change that behaviour ?

I've tested on runbot a case where I changed an internal transfer from Stock to Stock to WH to Stock (after reservation):

Move line:

![image](https://github.com/user-attachments/assets/f16581ed-056b-4965-adb3-dc239cfd26b9)


Move:

![image](https://github.com/user-attachments/assets/6d09a960-a30d-4235-9172-12f61170d498)


That is coherent.

Could you explain your use case ?

Reply:
This is the example use case we want to use.

https://github.com/user-attachments/assets/e4a9a25f-a002-4c39-805a-10113b88df1a